src/app/views/sign-in-view/sign-in-view.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="form-card">
+  <mat-card-title>Sign In</mat-card-title>
```

Comment:
Sugiero elegir hacer uno de los siguientes cambios:

- Cambiar todos (nombres de archivos, rutas, variables, etc.) los Sign In por Login (para ser consistente con el navbar), o
- Cambiar el Login del navbar por Sign in